reports/Alabama.md

```diff
@@ -1,3 +1,18 @@
+## Hoover
+
+### Police make a series of violent arrests | Date Unknown 
```

Comment:
The original tweet I found (see comment below) was posted 11:54 May 31st (local time in AL) so I think we could safely make this 31st May

```suggestion
### Police make a series of violent arrests | May 31st
```

Reply:
And thanks! The Doucette ones are a mess on dates